app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -13,7 +13,8 @@
         return $(e.target).val();
       }
 
-      return $('[name="order[payments_attributes][][payment_method_id]').val();
```

Comment:
hah! not only `"` is missing, but `"]` is missing!
your comment only ensures me this extraction is a good thing, its harder to make error closing all the braces